tensorflow/tensorboard/README.md

```diff
@@ -191,15 +191,6 @@ tutorial](https://www.tensorflow.org/versions/r0.10/how_tos/graph_viz/index.html
 
 # Frequently Asked Questions
 
-### Does TensorBoard support distributed summary writers?
```

Comment:
Can you please add:
"Violating the assumption that there is a single events file may cause TensorBoard to lose data or stop updating unexpectedly." to the other section, as this seems like an important thing to note.